### ISSUE TITLE
Editor's textarea cannot be initially navigated to

### ISSUE THREAD

**Issue by martintabakov** [telerik/kendo-ui-core#8281]
### Bug report

In the Editor's accessibility demo, you cannot initially navigate to the textarea using tab. After focusing the paragraph tool, pressing tab again will move the focus to the browser navigation bar. If you break the sequence and navigate to another tool before trying to navigate to the textarea, the issue disappears. The issue doesn't reproduce on the Keyboard Navigation demo.

### Reproduction of the problem

1. Open the Editor's accessibility demo - https://demos.telerik.com/kendo-ui/accessibility/editor
2. Try to navigate to the textarea using tab 

### Current behavior

The textarea will not be focused

### Expected/desired behavior

You should be able to navigate to the textarea on the first iteration
 

### Environment

* **Kendo UI version:** 2025.2.520
* **Browser:** [all]
